framework/bundles/org.jboss.tools.rsp.launching/META-INF/MANIFEST.MF

```diff
@@ -28,4 +29,5 @@ Export-Package: org.jboss.tools.rsp.eclipse.core.runtime,
  org.jboss.tools.rsp.launching,
  org.jboss.tools.rsp.launching.utils,
  org.jboss.tools.rsp.launching.internal;x-friends:="org.jboss.tools.rsp.server.test"
-Require-Bundle: org.jboss.tools.rsp.api;bundle-version="0.0.9"
+Require-Bundle: org.jboss.tools.rsp.api;bundle-version="0.0.9",
```

Comment:
require-bundle is an eclipse construct and we should not use it. We should only be using import-package.